data/src/main/java/org/cryptomator/data/cloud/pcloud/PCloudIdCloudNode.java

```diff
@@ -0,0 +1,9 @@
+package org.cryptomator.data.cloud.pcloud;
+
+import org.cryptomator.domain.CloudNode;
+
+interface PCloudIdCloudNode extends CloudNode {
```

Comment:
Is this intermedia interface really required? Can't we go straight to `PCloudNode extends CloudNode`?

Reply:
It's set up the same way as the `GoogleDriveIdCloudNode`.
This interface is directly used by `PCloudIdCache`.

Reply:
Let me rephrase the question: Why doesn't the `PCloudIdCache` just use `PCloudNode`?

Reply:
I mean I guess we could use `PCloudNode` directly but I'm not sure if I ran into some issues. I had it this way before.

Reply:
Ok, let me remove it. It doesn't seem to harm the cache.